app/controllers/users_controller.rb

```diff
@@ -21,6 +21,6 @@ def edit
   private
 
   def users_params
-    params.require(:user).permit(:name, :bio, :photo, :posts_counter, :email, :password)
+    params.require(:user).permit(:name, :bio, :photo, :posts_counter, :email, :password, :role)
```

Comment:
- [ ] OPTIONAL
It will be a good idea to define an admin action in your user controller, like below:

```ruby
...
   def admin?
     role == 'admin' ? true : false
   end
```